v0/17-10-0 Creating_report_templates.markdown

```diff
@@ -0,0 +1,37 @@
+# 17.10.0    Creating {{report}} templates
+
+A {{report}} template is a a way of saving a particular {{report}}, so saving a particular set of filters and display options, so that you can run them easily at any time.  These can be useful, for example, for regular reports for funders, so that you don't need to create a new report each time.  They are set-up and operate in the same way as a normal report, (but in a different place) and can then be chosen from the appropriate report menu.
```

Comment:
These can be useful, for example, for regular reports for funders, so that you don't need to create a new {{report}} each time.

In this sentence, the first report probably shouldn't be translated (and isn't) as it's "common" language; the second one should be as it refers to the functionality in Lamplight.  Other reports in this paragraph should probably have {{}}.